### ISSUE TITLE
Battle Logic

### ISSUE THREAD

**Issue by bowenbrinegar** [bowenbrinegar/pokemon-go#19]

**Comment by bowenbrinegar** [comment]
@kcgilland143 do you want to checkout the battlemode branch?

**Comment by kcgilland143** [comment]
Yeah! i've already checked it out. the hover branch includes all the commits from battlemode too. so I have been rebasing those commits with mine. so i have went through it all. i'm gonna make a commit to "hoverfunctionality" in a minute, we can merge that in, and then configuring the back-end shouldn't be too difficult

**Comment by bowenbrinegar** [comment]
Sweet, i'll keep an eye for it. I'm currently working on User Accounts
